### DISCUSSION TITLE
Windows Support on the README File

### DISCUSSION THREAD

**Discussion by RoyiAvital** [tshort/StaticCompiler.jl#171] [General]
The current README file [states](https://github.com/tshort/StaticCompiler.jl/blob/574db90bf816271129abf06e0371c2f8c00df844/README.md?plain=1#L81):

![image](https://github.com/user-attachments/assets/78de63ae-5ee5-4437-9140-d7289c2897a7)

> Doesn't work on Windows (but works in WSL on Windows 10+). PRs welcome.

Yet the 2 latest releases seem to add Windows support.

**Comment by brenhinkeller** [comment]
Ah yeah, I'll update that

**Reply by brenhinkeller** [reply]
Should be fixed by #172
